lib/Feature/FeatureModelParser.cpp

```diff
@@ -144,6 +169,11 @@ bool FeatureModelXmlParser::parseVm(xmlNode *Node) {
     FMB.setPath(Cnt ? fs::path(reinterpret_cast<char *>(Cnt.get()))
                     : fs::current_path());
   }
+  {
+    std::unique_ptr<xmlChar, void (*)(void *)> Cnt(
+        xmlGetProp(Node, XmlConstants::COMMIT), xmlFree);
+    FMB.setCommit(Cnt ? reinterpret_cast<char *>(Cnt.get()) : "");
```

Comment:
```suggestion
    FMB.setCommit(Cnt ? std::string(reinterpret_cast<char *>(Cnt.get())) : "");
```
This is a bit of a nitty gritty detail, currently is code is correct, however, if at some point would add an overload for `setCommit(char *)` we could have a use after free. So probably the save thing is to make the conversion to string explicit here.